apps/ledger-live-mobile/src/screens/WalletCentricAsset/AssetRow.tsx

```diff
@@ -36,6 +57,7 @@ const AssetRow = ({
       asset: currency.name,
     });
     if (navigationParams) {
+      // @ts-expect-error Ok, please stop doing this.
```

Comment:
todo : :fearful: maybe fix this if we know why we did this @nparigi-ledger @cgrellard-ledger @mlegall-ledger

Reply:
removed this part as it was unused, done